app/scripts/app.jsx

```diff
@@ -15,9 +15,11 @@ import NotFound from './components/static/not-found.jsx';
 React.render((
   <Router>
     <Route path="/" component={Root}>
-      <IndexRoute component={WaterPoints} />
-      <Route path="waterpoints/:id" component={WaterPoint} />
+      <Route path="waterpoints/" component={WaterPoints} >
+        <Route path=":id" component={WaterPoint} />
+      </Route>
       <Route path="data/" component={Data} />
+      <Route path="homepage/" component={Homepage} />
```

Comment:
I think this can be `<IndexRoute component={HomePage} />`, so that we don't have `/homepage/` in the URL